entrypoint.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-kapstan_api_base_url="https://api.kapstan.io/external"
+kapstan_api_base_url="https://api.kapstan.io/v2/external"
 filePath="/tmp/response.txt"
 
 # Function to trigger application deployment
 deployment_application() {
-  deployment_trigger_url="$kapstan_api_base_url/organizations/${INPUT_ORGANIZATION_ID}/workspaces/$INPUT_ENVIRONMENT_ID/applications/${INPUT_APPLICATION_ID}/deploy"
+  deployment_trigger_url="$kapstan_api_base_url/applications/${INPUT_APPLICATION_NAME}/deploy"
```

Comment:
have we tested the script against the case when application name consist spaces?

Reply:
We do not allow app names to have spaces. They can only have hyphens, alphabets and numbers.
Also verified that the script works on app-dev.
<img width="1512" alt="image" src="https://github.com/kapstan-io/deployment-action/assets/167970586/5351d8c1-489b-45fb-af30-99f20eab5a80">